Gemfile

```diff
@@ -66,7 +68,7 @@ end
 
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
-  gem "capybara"
-  gem "selenium-webdriver"
-  gem "webdrivers"
+  gem 'capybara'
+  gem 'selenium-webdriver'
+  gem 'webdrivers'
```

Comment:
it's okay doble quotes in gem, don't needs to change every where mannualy, Just try to use single quote in new code which you have added.